client/state/data-layer/wpcom/sites/home/layout/index.js

```diff
@@ -10,7 +10,6 @@ import config from '@automattic/calypso-config';
 
 const requestLayout = ( action ) => {
 	const isDev = config.isEnabled( 'home/layout-dev' ) || action.isDev;
-
```

Comment:
I kinda like this whitespace ;)

Reply:
Yeah same :)
I just made the PR using the auto revert button though 🤷